cmd/mungedocs/unversioned_warning.go

```diff
@@ -71,11 +76,15 @@ func updateUnversionedWarning(file string, mlines mungeLines) (mungeLines, error
 		// No warnings on release branches
 		return mlines, nil
 	}
+	linkToPreviousDoc := true
+	if hasLine(mlines, "<!-- TAG NEW -->") {
```

Comment:
fwiw, this is what we add for IS_VERSIONED:
```
<!-- BEGIN MUNGE: IS_VERSIONED -->
<!-- TAG IS_VERSIONED -->
<!-- END MUNGE: IS_VERSIONED -->
```
Going with that pattern, this should be
```
<!-- BEGIN MUNGE: IS_NEW -->
<!-- TAG IS_NEW -->
<!-- END MUNGE: IS_NEW -->
```